src/Microsoft.TemplateEngine.Core.Contracts/IOrchestrator.cs

```diff
@@ -3,29 +3,24 @@
 
 using System.Collections.Generic;
 using Microsoft.TemplateEngine.Abstractions;
-using Microsoft.TemplateEngine.Abstractions.Mount;
 
 namespace Microsoft.TemplateEngine.Core.Contracts
 {
```

Comment:
Are we fine with contract changes here? Technically Core.Contract should not be public - so probably yes. But I was wondering why we are keeping around unused methods (and actually the whole interface - I'd ideally drop it as a whole).